CHANGELOG.md

```diff
@@ -25,6 +25,9 @@ The changelog is available [on GitHub][2].
 * [#363](https://github.com/kowainik/summoner/issues/363):
   Move from `generic-deriving` to `generic-data`.
   (by [@chshersh](https://github.com/chshersh))
+* [#361](https://github.com/kowainik/summoner/issues/361):
+  Always put all warnings in `ghc-options` inside common stanza under `if`s.
```

Comment:
Let's add an example of `if` in here if you don't mind :slightly_smiling_face: 
Also, maybe makes sense to mention that the condition is on GHC in these `if`s? Or only one of these two options is fine as well, what do you think?